src/content/docs/mlops/bring-your-own/mlops-byo.mdx

```diff
@@ -1,18 +1,11 @@
 ---
 title: Bring your own data
-tags:
-  - MLOps integrations
-  - Byo
 metaDescription: "Bring your own ML model telemetry into New Relic, and start getting value from your ML model data in just a few minutes"
 redirects: 
   - /docs/alerts-applied-intelligence/mlops/bring-your-own/mlops-byo
 ---
 
-New Relic has entered the [MLOps](/docs/alerts-applied-intelligence/mlops/get-started/intro-mlops/) observability space, and is now allowing data scientists and machine learning engineers to monitor and observe the performance and effectiveness of their machine-learning models in a production environment.
-
-## Track your data and models in production easily [#byo]
-
-Once your production reaches a certain point in terms of the number of models it has to monitor, it becomes difficult to keep track of your data and model KPIs, resulting in even more complex systems. With **bring your own data (BYO)**, after just a few minutes, you can quickly send your model’s inference and metric data to the New Relic platform, directly from a notebook or any other environment.
+Once your production environment reaches a certain point in terms of the number of models it has to monitor, it becomes difficult to keep track of your data and model KPIs, resulting in even more complex systems. With **bring your own data (BYO)**, after just a few minutes, you can quickly send your model’s inference and metric data to the New Relic platform, directly from a notebook or any other environment.
```

Comment:
```suggestion
Once your production environment reaches a certain point in terms of the number of models it has to monitor, it becomes difficult to keep track of your data and model KPIs, resulting in even more complex systems. With **bring your own data (BYOD)**, after just a few minutes, you can quickly send your model’s inference and metric data to the New Relic platform, directly from a notebook or any other environment.
```

I don't see how it can't be BYOD...